modules/widgets/social-media-icons.php

```diff
@@ -109,7 +109,11 @@ public function widget( $args, $instance ) {
 		}
 
 		if ( ! empty( $instance['youtube_username'] ) ) {
-			$html .= '<li><a title="' . sprintf( $alt_text, esc_attr( $instance['youtube_username'] ), 'YouTube' ) . '" href="' . esc_url( 'https://www.youtube.com/channel/' . $instance['youtube_username'] ) . '" class="genericon genericon-youtube" target="_blank"><span class="screen-reader-text">' . sprintf( $alt_text, esc_html( $instance['youtube_username'] ), 'YouTube' ) . '</span></a></li>';
+			if ( substr( $instance['youtube_username'], 0, 2 ) == 'UC' ) {
```

Comment:
Actually ignore me.  I think this is fine - custom url's are usable in multiple ways: 
`youtube.com/customname`
`youtube.com/c/customname`
`youtube.com/user/customname`

So defaulting to `/user/` won't be a problem.